src/current/v24.1/changefeed-sinks.md

```diff
@@ -313,23 +317,38 @@ pubsub_sink_config = '{ "Flush": {"Messages": 100, "Frequency": "5s"}, "Retry":
 
 ### Pub/Sub sink messages
 
+Google Cloud Pub/Sub v2 is the default when emitting to `gcpubsub` sinks. In Pub/Sub v1, changefeeds with JSON encoders emitted events with the top-level message fields all lowercase:
+
+~~~
+{key: ..., value: ..., topic: ...}
+~~~
+
+In Pub/Sub v2, the top-level fields are capitalized:
+
+~~~
+{Key: ..., Value: ..., Topic: ...}
+~~~
+
+{{site.data.alerts.callout_danger}}
+After upgrading to CockroachDB v23.2 and later, changefeeds will emit to Pub/Sub v2 by default. You may need to reconfigure downstream systems to parse the new message format.
```

Comment:
Should we add a link to the cluster setting that users can toggle to change this behavior? Though if we do we may have to point out that the performance may be worse if you turn off the cluster setting.

Reply:
So, for v23.1 I have added the command because it is not on by default. For v23.2 and v24.1, there is a link to the cluster settings page that gives directions to the correct SQL. I assumed we'd really want users to keep this on. Also, the cluster setting is not `public` in 23.2 and 24.1, so it is not listed in the general settings list.

Reply:
Sounds good. Thanks!